cluster.go

```diff
@@ -244,6 +244,9 @@ type ClusterConfig struct {
 	// internal config for testing
 	disableControlConn bool
 	disableInit        bool
+
+	// If true tablet feature is enabled
+	experimentalTabletsEnabled bool
```

Comment:
I'm not sure if this is a right approach to making tablet support in the driver experimental. "Marking" it experimental has two reasons:
1. Make it clear that the API is unstable
2. Ensure that if newer versions of Scylla have a different tablet description that the driver will continue working correctly

As for point 2, it makes sense to make the code resilient to any issues while parsing the tablets, finding the tablets, making queries to `system.tablets` - no error should crash the driver.

As for point 1, I don't think that this flag really achieves the purpose of conveying that the API is unstable. However, I don't see a good way to do it in Go (there are build tags, but I don't think they work that well with libraries).

Reply:
Isn't the word "experimental" sufficient?

Maybe a comment? But no one reads those.